base.cfg

```diff
@@ -0,0 +1,122 @@
+[buildout]
+versions = versions
+parts =
+    instance
+    compileresources
```

Comment:
compileresources can be removed. is not used and probably will not be used in the near future